workshops/azure-spring-apps-enterprise/intermediate/01-workshop-environment-setup/README.md

```diff
@@ -14,16 +21,63 @@ This workshop leverages Github Codespaces to provide a development environment f
     az --version
     ```
 
-### Prepare your environment for deployments
+## Cloud Shell
+
+Login to Azure Portal and open Cloud Shell bash prompt
+
+![Alt text](../../../../media/cloudshell.png?raw=true "Optional Title")
+
+Make sure you have Bash Shell selected form the shell type dropdown
+
+![Alt text](../../../../media/bashshell.png?raw=true "Optional Title")
+
+## Git Bash
+
+1. Install Visual Studio code: [Download Visual Studio Code - Mac, Linux, Windows](https://code.visualstudio.com/download)
+2. Install Azure CLI: [Install the Azure CLI for Windows | Microsoft Learn](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli-windows?tabs=azure-cli)
+3. Install GIT: [Git (git-scm.com)](https://git-scm.com/)
+4. Install jq utility: [Download jq (jqlang.github.io)](https://jqlang.github.io/jq/download/)
+ Please note that you will have to rename jq-win64.exe to jq.exe and add it to PATH.
+5. Install Java 17: [Red Hat build of OpenJDK Download | Red Hat Developer](https://developers.redhat.com/products/openjdk/download?sc_cid=701f2000000RWTnAAO)
```

Comment:
We must use Microsoft Build of OpenJDK
https://www.microsoft.com/openjdk

See https://learn.microsoft.com/en-us/java/openjdk/download#openjdk-17